pwiz_tools/Skyline/Controls/Graphs/AreaCVToolbar.resx

```diff
@@ -126,16 +126,16 @@
   </data>
   <assembly alias="System.Drawing" name="System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
   <data name="toolStripLabel1.Size" type="System.Drawing.Size, System.Drawing">
-    <value>43, 28</value>

```

Comment:
Probably these sizing changes should all be reverted. It seems like you are forgetting to use only 100% scaling when editing forms in the designer.

Reply:
Reverted all changes to the AreaCVToolbar